README.md

```diff
@@ -454,6 +454,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [soegen](https://github.com/Ragmaanir/soegen) - Elasticsearch client for Crystal similar to the stretcher gem for ruby
 
 ## Serverless Computing
+ * [FaaStRuby](https://faastruby.io) - Serverless Software Development Platform for Ruby and Crystal.
```

Comment:
Cool. Fix the alpha order

Reply:
Done! I guess I should have named it _FaaStRuby 😝